constants/androidWebkitVersions.js

```diff
@@ -18,4 +18,5 @@ module.exports = {
   '4.2': '534.30',
   '4.3': '534.30',
   '4.4': '537.36',
+  '5.0': '537.36 (Chrome 37)'
```

Comment:
Is `(Chrome 37)` really a part of the webkit version?

Reply:
Yes, it is.